### DISCUSSION TITLE
`pairs` using Object.keys rather than Object.entries?

### DISCUSSION THREAD

**Discussion by Sawtaytoes** [ReactiveX/rxjs#6073] [General]
I noticed the `pairs` says it uses `Object.keys` instead of `Object.entries`. Is this still valid? If so, any reason it doesn't use `.entries`? Is it because of compatibly with older Node.js and browsers?

**Comment by cartant** [comment]
https://github.com/ReactiveX/rxjs/blob/c6ba9790ed42e4781278822aff96bce0863e7917/src/internal/observable/pairs.ts#L5-L7
